src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Figure extends Drawable {
```

Comment:
delete this interface

Reply:
> delete this interface

If i delete this interface program will be broken i use that interface in Main.java .